### ISSUE TITLE
[Bug]

### ISSUE THREAD

**Issue by mimsy55** [wind-addons/ElvUI_WindTools#465]
## To Reproduce

1. Go to 'group finder'
2. Click on 'premade dungeon click on dungeon groups"
1318x bad argument #1 to '?' (Usage: local activityInfo = C_LFGList.GetActivityInfoTable(activityID [, questID, showWarmode]))
[string "=[C]"]: in function `GetActivityInfoTable'
[string "@GroupFinderRio/GroupFinderRio.lua"]:883: in function <GroupFinderRio/GroupFinderRio.lua:878>
[string "=[C]"]: in function `LFGListUtil_SortApplicants'
[string "@Blizzard_GroupFinder/Mainline/LFGList.lua"]:1712: in function `LFGListApplicationViewer_UpdateResultList'
[string "@Blizzard_GroupFinder/Mainline/LFGList.lua"]:1517: in function `onEvent'
[string "@Blizzard_GroupFinder/Mainline/LFGList.lua"]:262: in function <...aceBlizzard_GroupFinder/Mainline/LFGList.lua:171>
[string "=[C]"]: in function `RefreshApplicants'
[string "@Blizzard_GroupFinder/Mainline/LFGList.lua"]:1548: in function <...aceBlizzard_GroupFinder/Mainline/LFGList.lua:1547>
[string "=[C]"]: ?
[string "=[C]"]: in function `Show'
[string "@Blizzard_GroupFinder/Mainline/LFGList.lua"]:308: in function <...aceBlizzard_GroupFinder/Mainline/LFGList.lua:303>
[string "=[C]"]: in function `LFGListFrame_SetActivePanel'
[string "@Blizzard_GroupFinder/Mainline/LFGList.lua"]:376: in function `LFGListFrame_FixPanelValid'
[string "@Blizzard_GroupFinder/Mainline/LFGList.lua"]:176: in function <...aceBlizzard_GroupFinder/Mainline/LFGList.lua:171>

Locals:
(*temporary) = "bad argument #1 to '?' (Usage: local activityInfo = C_LFGList.GetActivityInfoTable(activityID [, questID, showWarmode]))"
  windtools 3.8.7 
  elvui 13.81
wow build  11.0.7.58187
<img width="939" alt="22222222222222" src="https://github.com/user-attachments/assets/dcde11bd-efad-4f69-a81b-3b91e8eae324" />

**Comment by brooklynb7** [comment]
activityID format changes. Please change the code:

```lua
-- file: ElvUI_WindTools\Modules\Misc\LFGList.lua，line 282
local info = C_LFGList_GetActivityInfoTable(result.activityIDs[1])
```

**Comment by fang2hou** [comment]
fixed in 32082d946914b68601772087ac34d4ef4b287788
